tests/Benchmarks/Benchmarks.csproj

```diff
@@ -20,10 +21,10 @@
     <NoWarn>$(NoWarn);CS0618</NoWarn>
   </PropertyGroup>
   <ItemGroup>
+    <PackageReference Include="BenchmarkDotNet" Version="0.10.14.525" />
```

Comment:
nit: Add BenchmarkDotNetVersion property in dependencies.props and reference it here: https://github.com/dotnet/corefxlab/blob/master/tools/dependencies.props